tests/test_main.py

```diff
@@ -35,69 +33,69 @@
 def test_main_htseq(tmpdir):
     sample1 = str(datadir / Path("htseq") / Path("sample1.fragments_per_gene"))
     sample2 = str(datadir / Path("htseq") / Path("sample2.fragments_per_gene"))
-    expected_result = pd.DataFrame(data={
-        "feature": ["MSTRG.1", "MSTRG.2", "MSTRG.3", "MSTRG.4", "MSTRG.5",
-                    "MSTRG.6", "__alignment_not_unique", "__ambiguous",
-                    "__no_feature", "__not_aligned", "__too_low_aQual"],
-        "s1": [2371, 381, 741, 2361, 382, 706, 131, 2995, 0, 5, 0],
-        "s2": [0, 1, 7, 2, 3, 7, 13, 295, 0, 51, 0]
-    }, columns=["feature", "s1", "s2"])
+    expected_result = set([
```

Comment:
you could also use a set literal. `{1, 2, 3} == set([1, 2, 3])`. The literal is slightly faster than creating the list and making it a set. Not that speed matters. You might keep set for readability if you prefer that.